hail/src/main/scala/is/hail/check/Gen.scala

```diff
@@ -448,7 +448,7 @@ object Gen {
   }
 
   def interestingPosInt: Gen[Int] = oneOfGen(
-      oneOf(1, 2, Int.MaxValue - 1, Int.MaxValue),
+      oneOf(4, Int.MaxValue - 1, Int.MaxValue),
```

Comment:
4 seems like a pretty "typical" value already covered by `choose(1, 100)`. Isn't 1 a more interesting int we want to be sure to pick?

Reply:
1 is interesting, I suppose, but it leads to block matrices with tens of thousands of 1-byte partitions, which is slow :(

Reply:
oh, I changed this generally. I just need to change the BM gen.

Reply:
Ahh, that makes more sense!